CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+### Added
+- Added an [example](/examples/persistence-storage) of running a `kamu` API server along with a database,
```

Comment:
Is this better suited for `images/` directory? I would like `examples` to show usage patterns for the CLI. I think persistence is more like our internal testing concern rather than example at this stage.

Also mb name it `persistent-storage`? `persistence-storage` sounds like two synonyms.

Reply:
> Also mb name it persistent-storage? persistence-storage sounds like two synonyms.

thanks for pointing that out -- fixed

Reply:
> I would like examples to show usage patterns for the CLI
> Is this better suited for `images/` directory?

I get it, moved it to `images/` -- done